packages/jellyfish-transaction/__tests__/script/dftx/dftx_loans/CreateLoanScheme.test.ts

```diff
@@ -0,0 +1,60 @@
+import { SmartBuffer } from 'smart-buffer'
+import {
+  CCreateLoanScheme,
+  CreateLoanScheme
+} from '../../../../src/script/dftx/dftx_loans'
+import { OP_CODES } from '../../../../src/script'
+import { toBuffer, toOPCodes } from '../../../../src/script/_buffer'
+import { OP_DEFI_TX } from '../../../../src/script/dftx'
+import BigNumber from 'bignumber.js'
+
+it('should bi-directional buffer-object-buffer', () => {
+  const fixtures = [
+    '6a20446654784cc800000080b2e60e0000000006736368656d650000000000000000'
+  ]
+
+  fixtures.forEach(hex => {
+    const stack = toOPCodes(
+      SmartBuffer.fromBuffer(Buffer.from(hex, 'hex'))
+    )
+    const buffer = toBuffer(stack)
+    expect(buffer.toString('hex')).toStrictEqual(hex)
+    expect((stack[1] as OP_DEFI_TX).tx.type).toStrictEqual(0x4c)
+  })
+})
+
+const header = '6a20446654784c' // OP_RETURN, PUSH_DATA(44665478, 4c)
```

Comment:
add the correct descriptive comments?
https://github.com/DeFiCh/jellyfish/blob/f4ea2b399fe2ee799dedd4bb83ea8639e8178721/packages/jellyfish-transaction/__tests__/script/dftx/dftx_icxorderbook/ICXSubmitDFCHTLC.test.ts#L24

Reply:
Just changed. Hopefully, I add the correct comment